build.sbt

```diff
@@ -1534,6 +1534,59 @@ lazy val `engine-runner` = project
   .dependsOn(`polyglot-api`)
   .dependsOn(`logging-service`)
 
+// A workaround for https://github.com/oracle/graal/issues/4200 until we upgrade to GraalVM 22.x.
+// sqlite-jdbc jar is problematic and had to be exluded for the purposes of building a native
+// image of the runner.
+lazy val `engine-runner-native` = project
+  .in(file("engine/runner-native"))
```

Comment:
Creating new project is OK. Especially right now, when we need to tweak the dependencies & co.